website/package.json

```diff
@@ -24,6 +24,6 @@
     "d3-force": "^1.0.6",
     "d3-random": "^1.1.0",
     "react-virtualized": "^9.18.5",
-    "react-vis": "^1.9.2"
+    "react-vis": "^1.11.2"
```

Comment:
nice!  I wonder how this got out of date - perhaps bumping this should also be part of a script